inappmessaging/USERGUIDE.md

```diff
@@ -388,6 +388,9 @@ Documents targeting Product Managers:
 ## <a name="changelog"></a> Changelog
 
 ### 7.0.0 (in-progress)
+* SDKCF-4919: **Breaking Changes:** Build using Java 11.
```

Comment:
Please check if host app can no longer use our SDK if the app does not use Java 11

Reply:
The app can still run and receive the in-app messages when compiling/targeting java 8. I removed the Breaking change note.